### PR TITLE
Google Analytics Doc Update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated Google Analytics Extract doc to refresh screenshots and add a FAQ about compatible metrics-dimensions in GA with the appropriate reference link.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: cloud-extract-sources/google-analytics.mdx [hunk 1]
```diff
@@ -6,147 +6,107 @@ description: Step-by-step guide to ingest your Google Analytics data into Rudder
...
 
 <div class="warningBlock">
 
 The **Dimensions** and **Metrics** combination cannot be changed after the source is created and saved. The primary key that RudderStack creates depends on your choices during the source configuration.
-
 </div>
 
 <div class="infoBlock">
 
 If a combination of **Dimensions** and **Metrics** chosen by you is invalid, RudderStack will notify you.
```
Comment (ameypv-rudder) [inline cloud-extract-sources/google-analytics.mdx:59]: are we doing this? If no, I can remove this statement.

Comment (RanjeetMishra) [reply cloud-extract-sources/google-analytics.mdx:59]: > If a combination of **Dimensions** and **Metrics** chosen by you is invalid, RudderStack will notify you 
We do not have this support yet. Lets remove this.

**Review by RanjeetMishra — APPROVED** [review]
LGTM. Please remove the line that you mentioned in comment.

File: cloud-extract-sources/google-analytics.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply cloud-extract-sources/google-analytics.mdx]: Sounds good - removed!

**Pull request merged**